### ISSUE TITLE
Strange that native open-file-under-cursor is blocked

### ISSUE THREAD

**Issue by earthshrink** [vim-pandoc/vim-pandoc#376]
Consider the following contents in a file `bar.md`:
```
This is a link [foo](foo.html)
foo
```
and say both files `foo.md` and `foo.html` are present in the current directory. Editing `bar.md` in Vim, command `gf` on the link opens `foo.md` as advertised. But there is an issue if I try to use it in a more personal way.
If you place the cursor on the second line, `gf` does nothing, whereas Vim of itself would have opened `foo.md`, as you can check by temporarily renaming `bar.md` to `bar.txt`. This seems to break an existing capability with no obvious workaround.
I could have notes in text files keyed on words in my document, e.g., employee names, but `vim-pandoc` blocks my access to them in Vim, unless I bless every one of them into a markdown link.
Then there is #283 - the first link gets opened regardless of which one you `gf` or `gx` on. 
Maybe it is simpler to just throw away the hypertext module, and only set `sua`. Then entering `gf` or `gx` on any word or expanded link would just work as Vim intended.

**Comment by fmoralesc** [comment]
I find the normal `gf` behavior a bit suprising, I guess this goes to show how much I use `gf` and such. So TIL.

This is not well documented, but the current behaviour of overriding `gf` and `gx` can be disabled with

```{vim}
let g:pandoc#hypertext#use_default_mappings = 1
```
You are right tht #283 should be fixed, but the rationale for the module is to provide support for support for links that pandoc itself should understand (like relative links and labels `#sss`.) This will inevitably put it in conflict with the normal behavior:

```markdown
# Section 1 {#sec_one}

# Section 2 {#sec_one}

As we said in [section one](#sec_one)...
```

If you press over the link in section 2, you should move to section 1, not look for a file called `sec_one.md` in the current folder. Maybe the behaviour should be more complicated in that case. This idea should be extended also to other references, like LaTeX labels.

Maybe the better solution is to separate these concerns, and prefix the default mappings with `localleader`. Thoughts, @alerque?

**Comment by earthshrink** [comment]
@fmoralesc I tried the default mappings setting, did not restore the default behaviour. If I change the second line in `bar.md` to `foo.md`, then it gets opened regardless of the default mappings setting. Without explicit suffix `.md`, it does not open, regardless of the setting. Here's my .vimrc:
```
set sua+=.md
let g:pandoc#hypertext#use_default_mappings=1
```
I appreciate we need relative links to be logically complete, but this is broken. I did a `git  pull` before trying.
